app.js

```diff
@@ -98,43 +124,75 @@ app.action(
   },
 );
 
+app.action(
+  "start_help_form_crime",
+  async ({ body, action, ack, client, context }) => {
+    await ack();
+    await startHelpForm(client, body, "crime");
+  },
+);
+
 app.action(
   "start_help_form",
   async ({ body, action, ack, client, context }) => {
     await ack();
-    await startHelpForm(client, body);
+    await startHelpForm(client, body, "other");
   },
 );
 
 app.action(
   "submit_initial_help_request",
   async ({ body, action, ack, client, context }) => {
     await ack();
-    await submitInitialHelpRequest(body, client, "initial");
+    await submitInitialHelpRequest(body, client, "initial", "area");
+  },
+);
+
+app.action(
+  "submit_initial_help_request_crime",
+  async ({ body, action, ack, client, context }) => {
+    await ack();
+    await submitInitialHelpRequest(body, client, "initial", "crime");
   },
 );
 
 app.action(
   "advance_from_knowledge_store",
   async ({ body, action, ack, client, context }) => {
     await ack();
-    await submitInitialHelpRequest(body, client, "knowledge_store");
+    await submitInitialHelpRequest(body, client, "knowledge_store", "other");
   },
 );
 
 app.action(
   "advance_from_related_issues",
   async ({ body, action, ack, client, context }) => {
     await ack();
-    await submitInitialHelpRequest(body, client, "related_issues");
+    await submitInitialHelpRequest(body, client, "related_issues", "other");
+  },
+);
+
+app.action(
+  "advance_from_related_issues_crime",
+  async ({ body, action, ack, client, context }) => {
+    await ack();
+    await submitInitialHelpRequest(body, client, "related_issues", "crime");
   },
 );
 
 app.action(
   "submit_help_request",
```

Comment:
Is it worth changing these sort of values to non_crime? To align with the new additions to determine crime vs non_crime

Reply:
I think ok, eventually we'll want to retire the crime ones, should be fine to keep as-is